docs/extended_api/synchronization_primitives/barrier.md

```diff
@@ -21,8 +21,9 @@ It has the same interface and semantics as [`cuda::std::barrier`], with the
 
 ## Barrier Operations
 
-| [`cuda::barrier::init`]                 | Initialize a `cuda::barrier`. `(friend function)`                 |
-| [`cuda::device::barrier_native_handle`] | Get the native handle to a `cuda::barrier`. `(function template)` |
+| [`cuda::barrier::init`]                        | Initialize a `cuda::barrier`. `(friend function)`                 |
+| [`cuda::device::barrier_native_handle`]        | Get the native handle to a `cuda::barrier`. `(function template)` |
+| [`cuda::barrier::wait_parity/try_wait_parity`] | Wait on a `specific` phase of the barrier |
```

Comment:
I don't like this phrasing. @ogiroux how would you feel about "Wait on a phase whose parity matches the argument." or some such?